core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -520,7 +520,7 @@ private String transformOMAScore( String omaScore) {
         } else if( omaScore.equalsIgnoreCase("N") || omaScore.equalsIgnoreCase("neutral")) {
             return "N";
         } else if( omaScore.equalsIgnoreCase("[sent]")) {
-            return ExtendedMutationUtil.NOT_AVAILABLE; // TODO temp workaround for invalid sent values
+            return "NA"; // TODO temp workaround for invalid sent values
```

Comment:
@zheins what are these OMA scores? They don't seem to be in DB...maybe just remove this code??